httpd/httpd_test.go

```diff
@@ -11090,6 +11092,7 @@ func TestWebUserUpdateMock(t *testing.T) {
 	user.AdditionalInfo = "new additional info"
 	form := make(url.Values)
 	form.Set("username", user.Username)
+	form.Set("email", user.Email)
```

Comment:
please add a check to verify that the email was really set, and do the same also if the email is set using the REST API